geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/partitions/AttributeScheme.scala

```diff
@@ -89,6 +91,7 @@ object AttributeScheme {
 
   object Config {
     val AttributeOpt: String = "partitioned-attribute"
+    val EnumerationOpt: String = "enums"
```

Comment:
I think this could have a more descriptive name

Reply:
maybe `allow-list`? and could rename enums to `allowedValues`